File: PRACTICA/PARES/COLUMNAS/COLUMNAS.cpp
#include <iostream>
using namespace std;
int main() {
    int matriz[3][3] = {{1,2,3},{4,5,6},{7,8,9}};
    int suma[3] = {0,0,0};
    for(int i=0; i<3; i++)
        for(int j=0; j<3; j++)
            suma[j] += matriz[i][j];
    cout << "Suma de columnas: ";
    for(int j=0; j<3; j++) cout << suma[j] << " ";
}

File: PRACTICA/PARES/COLUMNAS/COLUMNAS.java
public class columnas {
    public static void main(String[] args) {
        int[][] matriz = {{1,2,3},{4,5,6},{7,8,9}};
        int[] suma = new int[3];
        for (int i=0; i<3; i++) {
            for (int j=0; j<3; j++) {
                suma[j] += matriz[i][j];
            }
        }
        System.out.print("Suma de columnas: ");
        for (int x : suma) System.out.print(x + " ");
    }
}

